phasmophobia_evidence_v3/widget.css

```diff
@@ -131,7 +131,7 @@
 } */
 
 .animated-box {
-  border-radius: {{animatedBorderRadius}}px;
+  border-radius: {{borderRadius}}px;
```

Comment:
How are we going to let users know that their animated border radius is no longer the same value as it was before?

This is a breaking change if the user never configured their borderRadius to be the same as their animatedBorderRadius.

Reply:
This has been reverted.